src/test/java/api/ClientApiHelper.java

```diff
@@ -0,0 +1,24 @@
+package api;
+
+import io.restassured.http.ContentType;
+import io.restassured.specification.RequestSpecification;
+
+import static api.entities.Login.bearerToken;
+import static io.restassured.RestAssured.given;
+
+public class ClientApiHelper {
```

Comment:
Let's create 'utils' folder inside the 'api' folder and move this class to the 'utils' folder.

Reply:
Fixed